src/service/layers-impl.js

```diff
@@ -751,6 +751,9 @@ export class LayoutElement {
     // This might lead to a double tracking.
     if (!this.children_.includes(child)) {
       this.children_.push(child);
+      if (child.isLayer()) {
+        this.transfer_(child);
+      }
```

Comment:
Is this an unrelated bug fix?

Reply:
It's an optimization that allows the parent layer to be discovered during `remeasure` calls (which are obviously measure phases) instead of when declaring an layout is a layer (which may be in a mutate phase).

Discovering the parent layer requires a measure phase because we have to check if an ancestor (or the element itself) has `position: fixed` style.